lerna.json

```diff
@@ -1,5 +1,5 @@
 {
   "version": "independent",
   "npmClient": "yarn",
-  "packages": ["packages/*"]
+  "packages": ["packages/*", "packages/draft/*"]
```

Comment:
New structure
```
- packages
  - component-library
  - draft
    - well
    - other draft components
```
IMO I think it's clearer keeping `draft` within packages instead of creating another root level folder. Thoughts?

Reply:
Agree, I think it's a good idea to imagine what it'll look like when it scales. Without your proposed structure it'll be `component-library` with like 50 siblings.

Reply:
Another option would be a sibling of packages, e.g.:

```yaml
# packages which define individual components
- components:
  - draft-...
  - draft-...
  - maybe-non-draft-also-at-some-point

# tokens, libraries of components, utilities, etc.
- packages:
  - component-library
  - tokens
```

Reply:
🤔 That might cause some confusion for newcomers looking for what components they should be using now. How about:
```
# Legacy or non-standard packages, and draft components
- packages:
  - deprecated-component-library-helpers
  - draft-well
  - maybe-non-draft-also-at-some-point

# Source of truth and current tokens, libraries of components, utilities, etc.
- core:
  - component-library
  - tokens
```

Reply:
I like the idea of separating the core packages from the "non core" packages.

Yeah, as Ally mentions, the non-core package aren't always components (like the  `deprecated-component-library-helpers`)

The only thing that’s slightly confusing to me is that `component-library` and `tokens` are packages, but the directory names makes it seem like they are not (might be confusing to people that aren’t familiar with the monorepo). I read "package" as "NPM packages" - they are all packages in that sense.

At the risk of being wordy, could it be `draft-packages` and `core-packages`?
(deprecated-component-library-helpers is not a draft per say, but it's in same spirit and I can't think of a better name).

Or is there something special about the name `packages` that Lerna requries?